src/cpp/statistics/types/typesPubSubTypes.cxx

```diff
@@ -1279,9 +1279,9 @@ namespace eprosima {
             Entity2LocatorTrafficPubSubType::Entity2LocatorTrafficPubSubType()
             {
                 setName("eprosima::fastdds::statistics::Entity2LocatorTraffic");
-                auto type_size = Entity2LocatorTraffic::getMaxCdrSerializedSize();
-                type_size += eprosima::fastcdr::Cdr::alignment(type_size, 4); /* possible submessage alignment */
-                m_typeSize = static_cast<uint32_t>(type_size) + 4 /*encapsulation*/;
+                auto type_size = static_cast<uint32_t>(Entity2LocatorTraffic::getMaxCdrSerializedSize());
+                type_size += static_cast<uint32_t>(eprosima::fastcdr::Cdr::alignment(type_size, 4)); /* possible submessage alignment */
+                m_typeSize = type_size + 4 /*encapsulation*/;
```

Comment:
What was wrong in the original code? Method `eprosima::fastcdr::Cdr::alignment()` takes and returns a `size_t` ...

Reply:
Nothing, we both corrected the same warning in different ways. I will fetch yours.